tests/test_examples.py

```diff
@@ -100,3 +106,102 @@ def test_relative_binding():
             result = RABFEResult.from_mol(mol)
             assert isinstance(result.dG_bind, float)
             assert isinstance(result.dG_bind_err, float)
+
+
+def test_smc_biphenyl():
+    """run_smc_on_biphenyl.py with reasonable settings, and expect
+    * output in summary_smc_result_*.pkl
+    * no NaNs in accumulated log weights
+    * delta_f in ballpark of 0
+    """
+    config = dict(n_walkers=100, n_windows=100, n_md_steps=100)
+    cli_args = [f"--{key}={val}" for (key, val) in config.items()]
+
+    with temporary_working_dir() as temp_dir:
+        # expect running this script to write a summary_result_result_{uid}.pkl file
+        output_path = str(Path(temp_dir) / "summary_smc_result_*.pkl")
+        assert len(glob(output_path)) == 0
+        _ = run_example("run_smc_on_biphenyl.py", cli_args, cwd=temp_dir)
+        smc_result_fnames = glob(output_path)
+        assert len(smc_result_fnames) == 1
+
+        # load result
+        with open(smc_result_fnames[0], "rb") as f:
+            smc_result = load(f)
+
+        # expect no NaNs in accumulated log weights
+        log_weights_traj = smc_result["log_weights_traj"]
```

Comment:
incremental_log_weights_traj, is log_weights_traj still being stored?

Reply:
I think this is correct as-is.

Confirmed `log_weights_traj` is being stored https://github.com/proteneer/timemachine/blob/4b914d94f04a3f1cfabb8da6ceaca8497c59e635/timemachine/md/smc.py#L105 , and I think it's slightly more stringent to check for nans in the accumulated log weights rather than the incremental log weights.

Reply:
Thanks for catching this -- fixed in https://github.com/proteneer/timemachine/pull/718/commits/7879e6768798dc8fab6f8dc0bb44bac35725e776